### ISSUE TITLE
WCAG 1.1.1: Images must have alternate text (img)

### ISSUE THREAD

**Issue by OmgImAlexis** [unraid/webgui#544]
#### Issue
`Images must have alternate text` ([image-alt](https://dequeuniversity.com/rules/axe/3.3/image-alt?application=msftAI))
#### Target application
[Tower/Dashboard](http://tower.local/Dashboard)
#### Element path
#vm-57f356ed-2b31-9509-75ac-9945d4fea617 > img
#### Snippet
`<img src="/plugins/dynamix.vm.manager/templates/images/windows.png" class="img">`
#### How to fix
    Fix any of the following:
      Element does not have an alt attribute
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute or the title attribute is empty
      Element's default semantics were not overridden with role="presentation"
      Element's default semantics were not overridden with role="none"
#### Environment
Chrome version 78.0.3904.87
This accessibility issue was found using Accessibility Insights for Web 2.10.3 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
